dex/market.go

```diff
@@ -53,6 +59,12 @@ func NewMarketInfo(base, quote uint32, lotSize, rateStep, epochDuration uint64,
 	if err != nil {
 		return nil, err
 	}
+
+	// Check for sensible epoch duration.
+	if epochDuration <= 0 {
```

Comment:
nit: uint64 cannot be < 0

Reply:
> nit: uint64 cannot be < 0

You mean to change the comment, right?

Reply:
He means you only need to check for `== 0`.